src/components/central-program.js

```diff
@@ -41,73 +53,77 @@ const CentralProgram= ({ data }) => {
                 </RequireWideScreen>
             }
             */}
-            <Container>
-                <Row>
-                    <Col md={9} xl={6} className="mx-auto">
-                        <div className="pt-4 pt-sm-4">
-                            <h2 className="pb-3 pt-sm-3 pt-md-2">{`Staff Roles (${data.site.siteMetadata.latestSchoolYear})`}</h2>
-                            <StaffRolesTable data={centralProgram.staff_roles} />
-                        </div>
-                        <div className="pt-4">
-                            <h2 className="pb-3">{`Staff Labor Unions (${data.site.siteMetadata.latestSchoolYear})`}</h2>
-                            <StaffLaborUnionsChart data={centralProgram.staff_bargaining_units}/>
-                            <StaffLaborUnionsTable data={centralProgram.staff_bargaining_units} />
-                        </div>
-                    </Col>
-                </Row>
-            </Container>
-        </div>
-        </Layout>
-    )
+        <Container>
+          <Row>
+            <Col md={9} xl={6} className="mx-auto">
+              <div id={`${ELEMENT_NAME_PREFIX}-1`} className="pt-4 pt-sm-4">
+                <h2 className="pb-3 pt-sm-3 pt-md-2">{`Staff Roles (${data.site.siteMetadata.latestSchoolYear})`}</h2>
+                <StaffRolesTable data={centralProgram.staff_roles} />
+              </div>
+              <div id={`${ELEMENT_NAME_PREFIX}-2`} className="pt-4">
```

Comment:
so if we add a new section, we just give it an id like this and up the count here and the total in `numSections`?

Reply:
Yep! It's a little annoying overhead to make sure they are consistent, but I couldn't think of a better way to allow `ScrollWidget` to be a separate component, and it felt a little heavy to define it within `central-program.js`.